CHANGELOG

```diff
@@ -1,5 +1,8 @@
 # CHANGELOG
 
+- 0.12.4.2 (TO BE RELEASED)
```

Comment:
We're adding functionality so this should be a minor version bump -- so `0.12.5.0`